public/_data.json

```diff
@@ -40,12 +40,12 @@
             "url": "https://www.fullstackreact.com/"
           },
           {
-            "name": "Google",
-            "logo": "host-google.png",
-            "url": "https://careers.google.com/locations/los-angeles/"
+            "name": "Philosophie",
+            "logo": "host-philosophie.png",
+            "url": "https://angel.co/philosophie/jobs"
           }
       ],
-      "titoUrl": "https://jsla-june-2017.eventbrite.com",
+      "titoUrl": "https://jsla-july-2017.eventbrite.com",
```

Comment:
we should use the affiliate link so we can track where RSVPs are coming from,
`https://www.eventbrite.com/e/july-2017-registration-36130641722?aff=site`